libs/shared/src/types/shared/index.ts

```diff
@@ -1 +1,5 @@
 export type CustomDataType = { [key: string]: string | string[] | boolean | number | undefined };
+
+export const NOVU_SUB_MASK = 'nvsk.';
```

Comment:
It's possible we'll add other forms of sub masks in the future, this provides avoidance of doubt and preparation for that future:
```suggestion
export const NOVU_ENCRYPTION_SUB_MASK = 'nvsk.';
```